### DISCUSSION TITLE
tmux broken on Mac

### DISCUSSION THREAD

**Discussion by moreleftpedal** [Homebrew/discussions#3368] [Everyday usage] [answered]
I’ve recently realized that tmux is not behaving correctly on all my Macs running the latest version of MacOS Monterey.  Basically the backspace key doesnt work moving the cursor only forward when hitting backspace and also it wont let you clear the screen and throws a ‘terminal database is inaccessible’.  

Installed version: tmux 3.3a

I have tried uninstalling and reinstalling to no avail.  Anyone have any ideas?  I’m dead in the water without tmux.  It works fine on my Ubuntu machines.  

Thanks in advance.

**Comment by SMillerDev** [comment]
Tmux now sets `tmux-256color` as the TERM. You might want to override that if you are using programs that use the macOS ncurses (since macOS comes with a very old ncurses)

**Reply by moreleftpedal** [reply]
Ahh interesting.  I didn't know that.  They must have just made that change recently.  What I don't understand is that terminal on Mac declares terminal as: 
<img width="671" alt="image" src="https://user-images.githubusercontent.com/107746699/174460390-3a4b1419-7c98-4e44-8168-4de4dfe5407b.png">

Does that not make a difference since the issue is with ncurses?  If so whats the solution then?  What do I need to do...sorry if this is obvious.

**Comment by carlocab** [comment]
You can either add
```
set -g default-terminal screen-256color
```
to your `~/.tmux.conf` or do
```
brew install zsh
```
and use Homebrew Zsh as your default terminal.

**Comment by moreleftpedal** [comment, marked as answer]
Yes that fixed it!  Thank you!  

Just clarity I had to create the `~/.tmux.conf` file because it didn't exist.  Once I created it and added `set -g default-terminal screen-256color` then everything was back to normal.  You rock!
